Source/Lib/Codec/EbThreads.c

```diff
@@ -84,7 +84,6 @@ EbHandle eb_vp9_create_thread(
         if (ret != 0) {
             if (ret == EPERM) {
 
-                pthread_cancel(*((pthread_t*)thread_handle));
                 free(thread_handle);
 
                 thread_handle = (pthread_t*)malloc(sizeof(pthread_t));
```

Comment:
unrelated: the above free and malloc could be skipped i think? but maybe good for clarify